index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Angel Uray Portfolio</title>
+    <title>Portfolio - Angel Uray</title>
```

Comment:
- [ optional] dont you think a better page title will be '`angel uray's portfolio'`